### DISCUSSION TITLE
Thank You and Upcoming GitHub Permissions Update

### DISCUSSION THREAD

**Discussion by jefftriplett** [djangopackages/djangopackages#1054] [General]
Hi everyone, 

This subject came up during the DjangoCon US 2023 sprints about how to clean up admin access and, in general, keep the project moving forward. 

I have also been concerned about the bus factor and getting the project's deployment in a more automated state. 

In that spirit, this is the draft I plan to post as an Issue here after getting some input to kick off the GitHub Admin cleanup. 

I'd love your thoughts, concerns, and questions before I post it.

----

Hello everyone 👋,

First and foremost, I'd like to express our deepest gratitude to everyone who has contributed to Django Packages over the years. Your dedication and effort have played a significant role in making this project what it is today. We sincerely appreciate all the time and energy you've invested here.

As part of our ongoing maintenance and to ensure the security of our repository, we're planning to update GitHub permissions at the end of the month on **Thursday, November 30, 2023**. This is a routine cleanup to manage access. Also, it might help some of our past contributors who no longer actively participate to stop receiving GitHub notifications.

## What does this mean for you?

- If you're still actively involved or wish to remain on the repository with the same level of access, please respond to this issue or contact us directly.
- If you've moved on to other projects and would prefer to have your permissions adjusted, you don't need to take any action; the update at the end of the month will take care of it.

This is purely an administrative action, and we hope it does not cause inconvenience. We're taking this step to ensure the best for our community and the security of our repository.

Thank you once again for being a part of our journey. We're looking forward to the continued growth and success of Django Packages with all your support!

Warm regards,
Jeff

**Comment by jefftriplett** [comment]
cc @saadmk11 and @ryancheley in case the discussion doesn't turn into a notification for you since you both have been more involved over the last year.

**Reply by ryancheley** [reply]
I’m all in on this 😁

**Reply by jefftriplett** [reply]
Thank you for the feedback. I'm going to convert this into a public issue.

**Comment by daheats** [comment]
@jefftriplett please keep me on. I am hoping that I can make more time for helping out in the upcoming months. ❤️

**Comment by jefftriplett** [comment]
This is now a GitHub Issue: https://github.com/djangopackages/djangopackages/issues/1058

**Comment by luzfcb** [comment]
@jefftriplett Although I won't be active for some time, I can help you with some DevOps tasks, such as keeping an eye on Sentry notifications (if any, as I currently don't have access) and other monitoring tools and fixing any problems that may arise.
Currently, I only have access to Digital Ocean.

If you act as a project manager, I might be able to fit some time a month into the opensource time window I have at @Labcodes
